### PR TITLE
Use well known location for bundled english content pack

### LINKED ISSUE #438: Use well-known location for bundled English content pack
## Summary

This is already done in the `ka-lite-bundle` Debian package. Basically, just install the content pack as described in the release notes:

> Dev Auto-discovery of content-packs in well-known location `/usr/share/kalite/preseed/contentpack-<version>.<lang>.zip`, example: `/usr/share/kalite/preseed/contentpack-0.17.en.zip`. Happens during `kalite.distributed.management.commands.setup`.

...it will also install the preseeded content pack automatically when running KA Lite for the first time, which will trigger `setup` to run automatically.

## Version

 - Installer: Windows/OSX
 - Version: 0.17.0

### PR DESCRIPTION
**Pull request by mrpau-richard**
## Summary

> Use well known location for bundled english content pack.
> Fix build script, step arrangements.

## Issues addressed

Fixes #438 

/cc @cpauya @mrpau-eduard

### CODE REVIEW

**Comment by cpauya** [comment]
@mrpau-richard I still see a comment about `Pyrun` at https://github.com/learningequality/ka-lite-installers/pull/441/files#diff-632e66fabe1d6056f28c4739c768f4afL212 - can you clear that up?

**Comment by mrpau-richard** [comment]
@cpauya OK I will remove it.

**Comment by mrpau-richard** [comment]
@cpauya I've fixed all of your comments. Can you check it again? Thanks

File: osx/build.sh [hunk 1]
```diff
@@ -177,12 +176,35 @@ test ! -d "$TEMP_OUTPUT_PATH" && mkdir "$TEMP_OUTPUT_PATH"
...
 
+KA_LITE_PROJECT_DIR="$SCRIPTPATH/KA-Lite"
+DMG_PATH="$OUTPUT_PATH/KA-Lite-Installer.dmg"
+DMG_BUILDER_PATH="$WORKING_DIR/create-dmg"
+CREATE_DMG="$DMG_BUILDER_PATH/create-dmg"
+KA_LITE_ICNS_PATH="$KA_LITE_PROJECT_DIR/KA-Lite/Resources/images/ka-lite.icns"
+
+test ! -d "$OUTPUT_PATH" && mkdir "$OUTPUT_PATH"
+
+((STEP++))
```
Comment (cpauya) [inline osx/build.sh]: Just a nitpick, you need to make an echo every time you increment the steps to give the user a feedback.  

With these changes, the user is notified only if the directory is non-existent.  If it exists already, the user won't know and will be shown a skipped step.

To simplify, just show an echo of the step description.

**Comment by benjaoming** [comment]
Was this released for the 0.17.0 installer?

I think it looks like @cpauya 's last review comment has been addressed, so is it good to merge?

**Comment by cpauya** [comment]
Oh sorry, yes @benjaoming this should be good to merge... doing that now.

**Comment by benjaoming** [comment]
Great.. but just to be clear:

> Was this released for the 0.17.0 installer?

**Comment by mrpau-richard** [comment]
Hi @benjaoming yeah, it's for 0.17 released fixes.

**Comment by cpauya** [comment]
But I don't see the changes on the `develop` branch - can you confirm @mrpau-richard if you have other branches with the fixes that we have released for 0.17.0?

**Comment by cpauya** [comment]
Well now that I've pulled the recent changes... it's on the `develop` branch already since I merged it.  But to be sure, the changes on this PR weren't included on the already released 0.17.0.  Is that correct @mrpau-richard?

**Comment by mrpau-richard** [comment]
@cpauya Yeah the changes of this PR was not in current 0.17 released.

**Comment by cpauya** [comment]
Thanks for confirming.

**Pull request merged**